packages/lab/src/tabs-next/TabNext.tsx

```diff
@@ -0,0 +1,78 @@
+import { makePrefixer } from "@salt-ds/core";
+import {
+  forwardRef,
+  ForwardedRef,
+  MouseEvent,
+  ReactElement,
+  useCallback,
+} from "react";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { TabProps } from "./TabsNextTypes";
+
+import tabCss from "./TabNext.css";
+import clsx from "clsx";
+
+const withBaseName = makePrefixer("saltTabNext");
+
+export const TabNext = forwardRef(function Tab(
+  {
+    ariaControls,
+    children,
+    className,
+    index,
+    value,
+    label,
+    onClick,
+    onKeyDown,
+    onKeyUp,
+    selected,
+    tabIndex,
+    ...props
+  }: TabProps,
+  ref: ForwardedRef<HTMLDivElement>
+): ReactElement<TabProps> {
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-tab-next",
+    css: tabCss,
+    window: targetWindow,
+  });
+  if (index === undefined || onClick === undefined || onKeyDown === undefined) {
+    throw Error(
```

Comment:
I thought it's `throw new Error`? Add a test maybe?

Reply:
`throw Error` works as well.